core/frontend/src/tile/map/MapLayerSources.ts

```diff
@@ -26,6 +26,10 @@ export enum MapLayerSourceStatus {
   InvalidCredentials,
   /** Provided format id could not be resolved in [[MapLayerFormatRegistry]] */
   InvalidFormat,
+  /** Format is not compatible with the URL provided.
+   * @beta
```

Comment:
What's the motivation for marking this `@beta`?

Reply:
No real motivation other than have the possibility to drop it if we change our mind.

Reply:
Do you think that's likely? If so resolve this comment.